src/components/Button.jsx

```diff
@@ -1,22 +1,29 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-function Button({ name }) {
+function Button({ name, handleClick }) {
   const symbols = name === '÷' || name === '*' || name === '-' || name === '+' || name === '=' ? 'orange' : '';
-  const bigN = name === 0 ? 'bigN' : '';
+  const bigN = name === '0' ? 'bigN' : '';
   return (
-    <button type="button" className={`${symbols} ${bigN}`}>
+    <button
+      type="button"
+      className={`${symbols} ${bigN}`}
+      onClick={() => {
+        handleClick(name);
+      }}
+    >
       {name}
     </button>
   );
 }
 
 Button.propTypes = {
-  name: PropTypes.string,
+  name: PropTypes.string.isRequired,
+  handleClick: PropTypes.func.isRequired,
 };
 
-Button.defaultProps = {
-  name: '',
-};
+// Button.defaultProps = {
+//   name: '',
+// };
```

Comment:
- [ ] Please remove commented code. This is not a good practice in writing clean code.